internal/wrappers/headers/realtime.hpp

```diff
@@ -20,13 +20,12 @@ namespace kuzzleio {
       Realtime(Kuzzle* kuzzle);
       Realtime(Kuzzle* kuzzle, realtime* realtime);
       virtual ~Realtime();
-      int count(const std::string& index, const std::string collection, const std::string roomId) Kuz_Throw_KuzzleException;
-      void join(const std::string& index, const std::string collection, const std::string roomId, room_options* options, NotificationListener* cb) Kuz_Throw_KuzzleException;
-      std::string list(const std::string& index, const std::string collection) Kuz_Throw_KuzzleException;
-      void publish(const std::string& index, const std::string collection, const std::string body) Kuz_Throw_KuzzleException;
-      std::string subscribe(const std::string& index, const std::string collection, const std::string body, NotificationListener* cb, room_options* options=NULL) Kuz_Throw_KuzzleException;
-      void unsubscribe(const std::string& roomId) Kuz_Throw_KuzzleException;
-      bool validate(const std::string& index, const std::string collection, const std::string body) Kuz_Throw_KuzzleException;
+      int count(const std::string& index, const std::string& collection, const std::string& roomId, query_options *options=NULL) Kuz_Throw_KuzzleException;
```

Comment:
It's really nitpicking level professional but I wonder why we are using the macro `NULL` instead of `nullptr` ?

Reply:
because we are not using C++11